integration test is not running

### What happened?

How to reproduce (on a GPU node)

```
./scripts/actions.sh integration-test
```

There are 2 issues:

A small one: `--offline` missing in the uv command

```
error: Failed to generate package metadata for `flash-attn==2.7.4.post1 @ direct+https://github.com/Dao-AILab/flash-attention/releases/download/v2.7.4.post1/flash_attn-2.7.4.post1+cu12torch2.6cxx11abiFALSE-cp312-cp312-linux_x86_64.whl`
  Caused by: Failed to fetch: `https://github.com/Dao-AILab/flash-attention/releases/download/v2.7.4.post1/flash_attn-2.7.4.post1+cu12torch2.6cxx11abiFALSE-cp312-cp312-linux_x86_64.whl`
  Caused by: Request failed after 3 retries
  Caused by: error sending request for url (https://github.com/Dao-AILab/flash-attention/releases/download/v2.7.4.post1/flash_attn-2.7.4.post1+cu12torch2.6cxx11abiFALSE-cp312-cp312-linux_x86_64.whl)
  Caused by: client error (Connect)
  Caused by: tcp connect error
  Caused by: Network is unreachable (os error 101)
```

A bigger one, as a consequence of the hackathon: the initialization of the loggers is incorrect

```
src/weathergen/utils/logger.py:130: in init_loggers
    logger = init_logger_per_stream(logger, min(streams), stream_handle)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

logger = <Logger weathergen (DEBUG)>, stream_handle = 20
output_streams = <_io.TextIOWrapper name="<_io.FileIO name=8 mode='rb+' closefd=True>" mode='r+' encoding='utf-8'>

    def init_logger_per_stream(logger, stream_handle, output_streams):
        for ostr in output_streams if type(output_streams) is tuple else [output_streams]:
            # determine correct stream handler
            with_color = True
            if getattr(ostr, "name", None) == "<stdout>" or getattr(ostr, "name", None) == "<stderr>":
                handler = logging.StreamHandler(ostr)
            elif ostr == "null":
                handler = logging.NullHandler()
            else:
                # path + file are specified as string or already path object
>               assert type(ostr) is str or type(ostr) is pathlib.Path
E               AssertionError

src/weathergen/utils/logger.py:56: AssertionError
```



### What are the steps to reproduce the bug?

see above

### Version

all

### Platform (OS and architecture)

all

### Relevant log output

```shell

```

### Accompanying data

_No response_

### Organisation

_No response_

--- a/scripts/actions.sh
+++ b/scripts/actions.sh
@@ -26,7 +26,7 @@ case "$1" in
   integration-test)
     (
       cd "$SCRIPT_DIR" || exit 1
-      uv run pytest ./integration_tests/small1_test.py --verbose
+      srun uv run pytest ./integration_tests/small1_test.py --verbose
     )
     ;;
   create-links)

--- a/scripts/actions.sh
+++ b/scripts/actions.sh
@@ -26,7 +26,7 @@ case "$1" in
   integration-test)
     (
       cd "$SCRIPT_DIR" || exit 1
-      srun uv run pytest ./integration_tests/small1_test.py --verbose
+      srun uv run --offline pytest ./integration_tests/small1_test.py --verbose
     )
     ;;
   create-links)

